client/db/bolt/db.go

```diff
@@ -95,7 +96,8 @@ var (
 	fromVersionKey         = []byte("fromVersion")
 	toVersionKey           = []byte("toVersion")
 	optionsKey             = []byte("options")
-	reservesKey            = []byte("reservesKey")
```

Comment:
For the record, I'm on board with breaking compatibility with the last dev version of the DB encoding.  Not worth an upgrade esp. considering the state of ETH (not used in the wild).